doc/swagger.json

```diff
@@ -5,14 +5,99 @@
         "version": "0.1"
     },
     "paths": {
+        "/taoPublishing/api/deliveries/publish": {
+            "post": {
```

Comment:
Could you please add request parameters description

Reply:
Thank you for pointing it out! I've changed the implementation a bit, parameters are expected not as query parameters, but as part of the body, the documentation updated as well.